### PR TITLE
Fixed: The bug and did responsive

### PR DESCRIPTION
**Pull request by InnoraftxAmarta**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,22 +1,109 @@
...
-//  var_dump($response);
-curl_close($curl);
+$response = $client->request("GET","https://ir-dev-d9.innoraft-sites.com/jsonapi/node/services");
 
+$res = $response->getBody();
 
+$res = json_decode($res);
 
-echo $curl;
+// var_dump($res);
```
Comment (kunal-sin) [inline index.php]: Why you have commented this code, if not required then plz remove.

File: index.php [hunk 2]
```diff
@@ -1,22 +1,109 @@
...
-?>
\ No newline at end of file
+$data = $res->data[$i]->relationships->field_image->links->related->href;
+
+// $client = new Client(['base_uri' => 'https://ir-dev-d9.innoraft-sites.com/jsonapi/node/services']);
+
+// var_dump($data);
+
+// echo $data;
+
```
Comment (kunal-sin) [inline index.php]: Same here

File: index.php [hunk 3]
```diff
@@ -1,22 +1,109 @@
...
+$image = $client->request("GET","$data");
```
Comment (kunal-sin) [inline index.php]: Why we are adding extra quotes in $data?

File: index.php [hunk 4]
```diff
@@ -1,22 +1,109 @@
...
+
+$imgres = $image->getBody();
+
+
+$imgres = json_decode($imgres);
+
+// var_dump($imgres);
```
Comment (kunal-sin) [inline index.php]: Same

File: index.php [hunk 5]
```diff
@@ -1,22 +1,109 @@
...
+
+$title = $res->data[$i]->attributes->field_secondary_title->value;
+
+$dataword = $res->data[$i]->attributes->field_services->value;
+
+$img = "https://ir-dev-d9.innoraft-sites.com".$imgres->data->attributes->uri->url;
+
+// var_dump($img);
+
+// echo $img;
```
Comment (kunal-sin) [inline index.php]: Same

File: index.php [hunk 6]
```diff
@@ -1,22 +1,109 @@
...
+    
+echo "<div class='col-xs-12 col-sm-12 col-md-6 col-lg-6 col-xl-4 col-xxl-4 image'>";
+
+echo "<div class='right text-break'>$title</div>";
+
+
+echo "<div class='para text-break'>$dataword</div>";
+
+echo "</div>";
+
```
Comment (kunal-sin) [inline index.php]: Please add proper indentation.

File: index.php [hunk 7]
```diff
@@ -1,22 +1,109 @@
...
+}
+
+
+// echo $response->getStatusCode(), "\n";
+
+// $arr = json_decode($response->getBody());
+
+// echo $arr;
+}
```
Comment (kunal-sin) [inline index.php]: Same here

File: index.php [hunk 8]
```diff
@@ -1,22 +1,109 @@
...
-$curl = curl_init();
+$client = new GuzzleHttp\Client;
 
-// curl_setopt_array($curl, array(
-//     CURLOPT_URL => "https://ir-dev-d9.innoraft-sites.com/jsonapi/node/services" ,
-//     CURLOPT_HTTPHEADER => array(
-//         "Content-Type: text/plain"
-//     ),
-// ));
+for($i=15;$i>=12;$i--){
```
Comment (kunal-sin) [inline index.php]: Why this is static?

**Comment by kunal-sin** [comment]
@InnoraftxAmarta Please don't use HTML in PHP tags

File: index.php (hunk 1 above)
Comment (InnoraftxAmarta) [reply index.php]: fixed

File: index.php (hunk 2 above)
Comment (InnoraftxAmarta) [reply index.php]: removed

File: index.php (hunk 3 above)
Comment (InnoraftxAmarta) [reply index.php]: removed quotes

File: index.php (hunk 4 above)
Comment (InnoraftxAmarta) [reply index.php]: removed comment

File: index.php (hunk 5 above)
Comment (InnoraftxAmarta) [reply index.php]: changed

File: index.php (hunk 6 above)
Comment (InnoraftxAmarta) [reply index.php]: Added proper indentation

File: index.php (hunk 7 above)
Comment (InnoraftxAmarta) [reply index.php]: removed

File: index.php (hunk 8 above)
Comment (InnoraftxAmarta) [reply index.php]: fixed

File: index.php [hunk 9]
```diff
@@ -1,22 +1,90 @@
...
-$curl = curl_init();
+$client = new GuzzleHttp\Client;
 
-// curl_setopt_array($curl, array(
-//     CURLOPT_URL => "https://ir-dev-d9.innoraft-sites.com/jsonapi/node/services" ,
-//     CURLOPT_HTTPHEADER => array(
-//         "Content-Type: text/plain"
-//     ),
-// ));
+for($i=15;$i>=0;$i--){
```
Comment (kunal-sin) [inline index.php]: This is still static, how do you know that data will remain 15 here?

File: index.php [hunk 10]
```diff
@@ -1,22 +1,90 @@
...
-//  var_dump($response);
-curl_close($curl);
+$response = $client->request("GET","https://ir-dev-d9.innoraft-sites.com/jsonapi/node/services");
 
+$res = $response->getBody();
 
+$res = json_decode($res);
 
-echo $curl;
+$data = $res->data[$i]->relationships->field_image->links->related->href;
```
Comment (kunal-sin) [inline index.php]: what if the response is empty?

File: index.php [hunk 11]
```diff
@@ -1,22 +1,90 @@
...
+$imgres = $image->getBody();
+
+
+$imgres = json_decode($imgres);
+
+$title = $res->data[$i]->attributes->field_secondary_title->value;
+
+$dataword = $res->data[$i]->attributes->field_services->value;
+
+$img = "https://ir-dev-d9.innoraft-sites.com".$imgres->data->attributes->uri->url;
```
Comment (kunal-sin) [inline index.php]: What if the response is empty?

File: index.php [hunk 12]
```diff
@@ -1,22 +1,90 @@
...
+
+<div class='right text-break'><?php echo $title; ?></div>
+
+
+<div class='para text-break'><?php echo $dataword; ?></div>
+
+</div>
+
+</div>
+
```
Comment (kunal-sin) [inline index.php]: I still don't see the proper indentation.

File: index.php [hunk 13]
```diff
@@ -1,22 +1,90 @@
...
+        
+<div class='col-xs-12 col-sm-12 col-md-6 col-lg-6 col-xl-4 col-xxl-4 image'>
+    
+<div class='right text-break'><?php echo $title; ?></div>
+    
+<div class='para text-break'><?php echo $dataword; ?></div>
+    
+</div>
+    
+</div>
```
Comment (kunal-sin) [inline index.php]: Indentation problem

Comment (InnoraftxAmarta) [reply index.php]: fixed Indentation problem

File: index.php (hunk 9 above)
Comment (InnoraftxAmarta) [reply index.php]: changed

File: index.php [hunk 14]
```diff
@@ -1,22 +1,89 @@
...
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+</head>
+<body>
+
 <?php
+require __DIR__ . '/vendor/autoload.php';
+
+$client = new GuzzleHttp\Client;
+
+for ($i = 15; $i >= 0; $i--) {
```
Comment (kunal-sin) [inline index.php]: This is still not dynamic. What if the size increases to 20?

Comment (InnoraftxAmarta) [reply index.php]: changed

File: index.php (hunk 11 above)
Comment (InnoraftxAmarta) [reply index.php]: then it will not go inside the loop

File: index.php (hunk 12 above)
Comment (InnoraftxAmarta) [reply index.php]: changed

File: index.php (hunk 10 above)
Comment (InnoraftxAmarta) [reply index.php]: it will not go inside the loop

Comment (kunal-sin) [reply index.php]: What if $response->getBody() is empty?

File: index.php (hunk 11 above)
Comment (kunal-sin) [reply index.php]: What if $image->getBody() is empty?

File: index.php (hunk 10 above)
Comment (InnoraftxAmarta) [reply index.php]: fixed

File: index.php (hunk 11 above)
Comment (InnoraftxAmarta) [reply index.php]: fixed $image->getBody() for being empty

**Comment by kunal-sin** [comment]
@InnoraftxAmarta Please merge this PR.

**Pull request merged**